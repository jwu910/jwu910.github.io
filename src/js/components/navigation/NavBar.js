import React from 'react';

import { Link } from 'react-router-dom';

const NavBar = (props) => (
  <ul>
    <li><Link to="/test" /></li>
  </ul>
);

export default NavBar;