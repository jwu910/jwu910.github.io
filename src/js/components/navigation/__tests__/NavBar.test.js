import React from 'react';
import renderer from 'react-test-renderer';

import { BrowserRouter as Router } from 'react-router-dom';

import NavBar from '../NavBar';

describe('Snapshots', () => {
  it('Renders correctly', () => {
    const component = renderer
      .create(
        <Router>
          <NavBar />
        </Router>
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});