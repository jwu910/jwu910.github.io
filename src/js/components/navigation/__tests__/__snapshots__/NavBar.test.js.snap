// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots Renders correctly 1`] = `
<div
  className="navbar-container"
>
  <div
    className="main-links"
  >
    <ul
      className="nav-list"
    >
      <li
        className="nav-list-item"
      >
        <a
          className="nav-link"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="nav-list-item"
      >
        <a
          className="nav-link"
          href="/portfolio"
          onClick={[Function]}
        >
          Portfolio
        </a>
      </li>
      <li
        className="nav-list-item"
      >
        <a
          className="nav-link"
          href="/contact"
          onClick={[Function]}
        >
          Contact
        </a>
      </li>
    </ul>
  </div>
</div>
`;
